doc/articles/get-started-wizard.md

```diff
@@ -1,4 +1,6 @@
-# Congratulations, you've just created a new project using the Uno Platform project templates
+# Welcome to Uno Platform!
```

Comment:
There's 2 `.md` links at the end of this file: they should be changed for `.html`.

Reply:
Why ? The docs tooling translates them automatically